API.md

```diff
@@ -157,7 +157,7 @@ Onyx.merge(ONYXKEYS.POLICY, {name: 'My Workspace'}); // -> {id: 1, name: 'My Wor
 ```
 <a name="mergeCollection"></a>
 
-## mergeCollection(collectionKey, collection)
+## mergeCollection(collectionKey, collection, mergeReplaceNullPatches)
```

Comment:
Should we really be exposing the merge replacement patches parameter to the external facing API? It seems like something we are really only planning to use within Onyx. And I can't think of why Onyx users would need this.

I see that we do pass this value to merge collection within onyx. I think we should make a private and internal version of the function to keep the external facing API the same.